src/EmailAddressValidator.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace ipl\Validator;
+
+use ipl\I18n\Translation;
+
+/**
+ * Validates Email address
+ */
+class EmailAddressValidator extends BaseValidator
+{
+    use Translation;
+
+    public function isValid($value)
+    {
+        // Multiple isValid() calls must not stack validation messages
+        $this->clearMessages();
+
+        // Validate email address
+        if (! filter_var($value, FILTER_VALIDATE_EMAIL)) {
```

Comment:
This only validates against [RFC822](http://www.faqs.org/rfcs/rfc822.html).

Zend's validator validates against its successor [RFC2822](http://www.faqs.org/rfcs/rfc2822.html).

Just because of this, I'd call this implementation a step backwards. Though, I didn't check what RFC2822 makes a better candidate to follow. Zend's validator is rather extensive, but I don't think RFC2822 replaces RFC822 so substantially that `FILTER_VALIDATE_EMAIL` doesn't apply anymore. I just think we should validate some of the additional things RFC2822 defines.

So, please identify those additional things. Then we can further discuss whether we should validate them or not.